kythe/go/extractors/config/extractor.go

```diff
@@ -29,21 +29,35 @@ import (
 // kytheConfigFileName The name of the Kythe extraction config
 const kytheExtractionConfigFile = ".kythe-extraction-config"
 
+// Repo is a loose container of input/output parameters for doing extraction on
+// remote repositories.
+type Repo struct {
+	// URI is the remote location of the repo itself.
```

Comment:
[optional] It's not essential to use the "starts with name" convention for struct fields (though it is fine if you prefer it).